netket/experimental/driver/tdvp_common.py

```diff
@@ -60,7 +62,8 @@ def __init__(
         self,
         operator: AbstractOperator,
         variational_state: VariationalState,
-        integrator: RKIntegratorConfig,
+        solver: AbstractSolver = None,
+        integrator: AbstractSolver = None,
         *,
         t0: float = 0.0,
```

Comment:
```suggestion
        solver: AbstractSolver = None,
        *,
        t0: float = 0.0,
        integrator: AbstractSolver = None,
```

we should warn only if integrator is explicitly specified as a keyword argument. 
We don't need to support it as a positional argument (it would be wrong to pass the solver twice...)